SECURITY-INSIGHTS.yml

```diff
@@ -0,0 +1,31 @@
+header:
+  schema-version: "1.0.0"
+  expiration-date: "2023-12-15T19:10:00.000Z"
```

Comment:
What's the expiry date used for?

Reply:
According to the [specification](https://github.com/ossf/security-insights-spec/blob/main/specification.md#specification), it's a required field. The description is:

> The date this file should no longer be considered valid, and it can be at most one year from the date it was last reviewed.

Reply:
Thanks, so should it be 2024 then? Otherwise it would be expired right now

Reply:
I'm sure there are some other fields I could have added, but I only wanted to add the ones I was fairly confident were correct.  I looked through the specification and at examples from other projects, so hopefully this is a good starting point.

Reply:
Good point!  I'll fix that.